Subject: Pptsetup: Do not mess up the password when reading from stdin
Origin: upstream, commit 1.10.0-4-gf449742
Author: Artur Juraszek <artur@juraszek.xyz>
Date: Wed Dec 4 01:22:12 2019 +0100

    Before this patch the trailing newline, which Perl by default leaves, had been
    included in the resulting configuration, thus having absolutely no chance to work
    without passing it via `--password`, which is *super insecure* due to being
    stored in shell's history and, quite often, showing up again in surprising ways.

--- a/pptpsetup
+++ b/pptpsetup
@@ -55,6 +55,7 @@
     if ( !$PASSWORD ) {
         print "Password: ";
         $PASSWORD = <STDIN>;
+        chomp $PASSWORD;
         $PASSWORD =~ s/([^\x20\x21\x23-\x7e])/sprintf ("\\x%02x", ord ($1))/eg;
     }
 
